CombinatorialGames/Game/Concrete.lean

```diff
@@ -44,7 +44,11 @@ structure ConcreteGame (α : Type v) : Type v where
   rightMoves : α → Set α
 
 namespace ConcreteGame
-variable {c : ConcreteGame.{v} α}
+variable {c : ConcreteGame.{v} α} {p : Player}
+
+variable (c p) in
+def moves : α → Set α :=
```

Comment:
This is missing a docstring (but I get that this is a temporary fix before #205, so I'll let it slide).